problems/2017-04-11-side-effects-may-include/tests/getYearDuration.spec.js

```diff
@@ -0,0 +1,11 @@
+const chai = require('chai');
+const expect = chai.expect;
+const sinon = require('sinon');
+
+const getYearDuration = require('../lib/getYearDuration').default;
+
+describe('getYearDuration', () => {
```

Comment:
True, I was banking on the comment in the module, but it doesn't hurt to have one in the test.